react/components/ProductImages/components/Carousel/index.js

```diff
@@ -35,6 +36,17 @@ const initialState = {
   gallerySwiper: null,
 }
 
+const CSS_HANDLES = [
```

Comment:
Nitpick: same thing as the other comment about `CSS_HANDLES` position on the file